### ISSUE TITLE
Cloud, je peux changer les chaines, c'est tout

### ISSUE THREAD

**Issue by Lodoxx** [Aymkdn/assistant-plugins#234]
Bonjour

Merci encore pour la travaille.
j'ai testé les applets qui m’intéresses, le résultat, c'est que je peux changer que les chaines, sinon, baisser le son, descendre, couper, baisser, augmenter, allumer la freebox, l’éteindre, impossible.
Une idée du problème ?

Merci d'avance.

**Comment by Aymkdn** [comment]
Tu as bien activé les applets pour chaque commande ? Google Home te répond correctement mais rien ne se passe sur la télé ?

**Comment by Lodoxx** [comment]
oui, j'ai les est activés à plusieurs reprise, rien n'y fait. Google Home me répond correctement, rien ne se passe sur la tv.

**Comment by Aymkdn** [comment]
Tu peux me donner un bout de ton e-mail que je puisse te retrouver et voir si je peux comprendre d'où ça vient. Je ne suis pas chez moi ce week-end donc limité à mon téléphone pour le moment...

**Comment by Lodoxx** [comment]
Pas de  problème: (..........sur31@gmail.com) . Merci d'avance.

**Comment by Aymkdn** [comment]
Alors je vois que en effet y'a que les commandes de zappe qui ont été reçues mais pas les autres. Je n'ai pas accès au serveur de mon téléphone donc je ne peux pas dans l'immédiat regarder plus en profondeur. Je regarde ça dès que je peux, mais pour le moment vérifie que tu as bien l'URL pour tes applets. Et je te recontacte sous peu.

**Comment by Lodoxx** [comment]
Bonjour

Je suis pas sur, mais je crois savoir: les seuls qui fonctionnent ce sont les zapping, ici:
https://assistant.kodono.info/freebox/ 

en (4) quand je vais pour copier : "zappe sur" & "Je zappe" 
et en (3) allume la Freebox et zappe sur & OK, j'allume la Freebox et zappe

ce sont les deux seul qui possèdent un "$" à la fin

Pour les autres je rajoute un $ dans Google Assistant, mais ça semble ne pas suffire

**Comment by Aymkdn** [comment]
Il faut copier mot pour mot ce qu'il y a dans le tutoriel. Le $ est un ingredient qui permet de remplacer quelque chose. Par exemple le mot "M6".

Les autres commandes n'ont donc pas besoin de ça. C'est pourquoi il faut suivre à la lettre les instructions du guide.
